src/components/Button.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable react/require-default-props */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = ({ name, color, wide }) => (
+  <button
+    className="button"
+    type="button"
+    style={{
+      backgroundColor: color ? '#dfdfdf' : null,
+      width: wide ? '50%' : null,
+    }}
+  >
+    {name}
+  </button>
+);
+
+Button.propTypes = {
+  name: PropTypes.string.isRequired,
+  color: PropTypes.bool,
+  wide: PropTypes.bool,
+};
+
+Button.defaultProps = {
+  color: false,
```

Comment:
You should not set the default color to false ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorProps.png)